src/main/java/frc/robot/commands/AutoBalancePID.java

```diff
@@ -13,17 +14,24 @@ public class AutoBalancePID  extends CommandBase {
     // Adds drive train components to balance command
     public AutoBalancePID(Drivetrain drivetrain) {
         m_drivetrain = drivetrain;
-        pid = new PIDController(.5, 0, 0);
+        pid = new PIDController( 0.6/10, 0, 0.02);
         pid.setTolerance(2);
         addRequirements(m_drivetrain);
-        pid.setSetpoint(0);
+        pid.setSetpoint(0); 
+
+       //SmartDashboard.putNumber("Auto Balance Max Output", 0.53);
+       //SmartDashboard.putNumber("Auto Balance P", 0.6/10);
+       //SmartDashboard.putNumber("Auto Balance D", 0.02);
 
     }
 
     @Override
     public void execute() {
-         double output = MathUtil.clamp(pid.calculate(m_drivetrain.getPitch(), 0), -0.4, 0.4);
-         m_drivetrain.arcadeDrive(-1*output,0);
+        double max_output = SmartDashboard.getNumber("Auto Balance Max Output", 0.53);
+        pid.setP(SmartDashboard.getNumber("Auto Balance P", 0.03));
+        pid.setD( SmartDashboard.getNumber("Auto Balance D", 0.004));
```

Comment:
Don't forget to update these default values according to the final values we ended with today!